python_ta/debug/snapshot.py

```diff
@@ -37,11 +37,25 @@ def get_filtered_global_variables(frame: FrameType) -> dict:
     return {"__main__": true_global_vars}
 
 
+def get_filtered_local_variables(frame, exclude_vars: Optional[Iterable[str | re.Pattern]]) -> dict:
+    """
+    Helper function for filtering local variables in a frame.
+    """
+    if exclude_vars:
+        return {
+            var: frame.f_locals[var]
+            for var in frame.f_locals
+            if not any(re.search(regex, var) for regex in exclude_vars)
+        }
+    return frame.f_locals
+
+
 def snapshot(
     save: bool = False,
     memory_viz_args: Optional[list[str]] = None,
     memory_viz_version: str = "latest",
     include: Optional[Iterable[str | re.Pattern]] = None,
```

Comment:
I think you're absolutely correct to rename this to `include_frames`, please go ahead to make that change in this PR.